helpdesk_mgmt_team_customer/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,60 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    partner_id_domain = fields.Binary(
+        help="This is the computed domain to filter partners.",
+        compute="_compute_partner_id_domain",
+    )
+
+    @api.depends(
+        "team_id",
+        "team_id.default_partner_id",
+        "team_id.allowed_partner_ids",
+        "team_id.is_unique_partner",
+    )
+    def _compute_partner_id_domain(self):
+        for record in self:
+            if record.team_id:
+                if record.team_id.default_partner_id:
```

Comment:
```suggestion
                if record.team_id.default_partner_id and not record.partner_id:
```
With this you can only select the default partner, and will give an error when saving. Try this.

Reply:
Fixed, Thanks!

Reply:
why dont't you use the field: commercial_partner_id?
When a contact from a company open a ticket, do you what to apply the restriction on the contact or on the company?

Reply:
The purpose of this feature is not to restrict which user/contact can open a ticket, but to restrict which partner can be assigned to the ticket depending on the helpdesk team.

In our use case, an internal user creates a ticket (or assigns an existing one) and then sets the team_id. Once the team is chosen:

the partner_id is auto-filled with the team’s default_partner_id, and the partner_id field is restricted to a specific list of contacts configured on the team (default_partner_id and allowed_partner_ids).

This behaviour is independent from the contact/company of the user who opens the ticket (portal or internal). We are configuring which partners are allowed for a given team, not which company the ticket creator belongs to.

For this reason I’m using partner_id directly instead of commercial_partner_id. In our scenario, a team can be dedicated to specific contacts, even within the same company.